Assets/ScreenSwipe/ScreenSwipe.cs

```diff
@@ -313,6 +313,7 @@ private void PaginationToggleCallback(bool isOn)
 			if (toggles[i].isOn)
 			{
 				GoToScreen(i);
+                if (onScreenChanged != null) onScreenChanged.Invoke(i);
```

Comment:
tried to use tabs, not sure if I did it correctly

Reply:
If you use Visual Studio my settings look like this:
![image](https://user-images.githubusercontent.com/8637789/57652693-3ca72d00-7624-11e9-84a0-1d3ba73c9eb7.png)
but I also use an extension [Format Document on Save](https://marketplace.visualstudio.com/items?itemName=mynkow.FormatdocumentonSave) so I don't have to manually format all the time :)

It looks like its using spaces still. Also, instead of use `!= null` you can do `onScreenChanged?.Invoke(i)`

Reply:
I have looked more into this and it seems `onScreenChanged` event aren't being invoked when Next/Prev buttons are pressed as well. It also, calls the `onScreenTweenEnd` callback twice, because the toggle changed events is also calling `GoToScreen()`

I'm not actually sure `onScreenTweenEnd` needs to exist at all really... I don't have it in my main project where this script drives from. So could probably be removed... `onScreenChanged` should be enough.

But never the less, I've added a test script to test the callbacks which will log in console.

Reply:
i used `!= null` over `?.Invoke` just to make it .Net 3.5 Compatible, no other reason

Reply:
ok fair enough. I forget about .NET 3.5, I always make every thing .NET 4.x xD